{"tag": "function", "depth": 1, "time": 1563354515.4577825, "data": {"name": "record_ui", "call_args": {"driver": "<poco.drivers.android.uiautomation.AndroidUiautomationPoco object at 0x0F06CC90>", "action": "'click'", "ui": "UIObjectProxy of \"text=\u56fe\u7247\u6807\u8bb0\"", "args": "[0.125, 0.25859375]"}, "start_time": 1563354515.4577825, "ret": "UIObjectProxy of \"text=\u56fe\u7247\u6807\u8bb0\"", "end_time": 1563354515.4577825}}
{"tag": "function", "depth": 2, "time": 1563354516.1571789, "data": {"name": "try_log_screen", "call_args": {"screen": null}, "start_time": 1563354515.792033, "traceback": "Traceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 40, in _install_and_setup\n    apk_path = self.adb.path_app(ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 1162, in path_app\n    raise AirtestError('package not found, output:[%s]' % output)\nairtest.core.error.AirtestError: 'package not found, output:[]'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\logwraper.py\", line 72, in wrapper\n    res = f(*args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\cv.py\", line 97, in try_log_screen\n    screen = G.DEVICE.snapshot()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\android.py\", line 217, in snapshot\n    self.rotation_watcher.get_ready()\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\snippet.py\", line 104, in ready_func\n    method()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 61, in start\n    self._install_and_setup()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 42, in _install_and_setup\n    self.adb.install_app(ROTATIONWATCHER_APK, ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 568, in install_app\n    out = self.cmd(cmds)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 185, in cmd\n    raise DeviceConnectionError(stderr)\nairtest.core.error.DeviceConnectionError: \"error: device '76UBBL422HJF' not found\\r\\nerror: device '76UBBL422HJF' not found\\r\\nadb: connect error for write: device '76UBBL422HJF' not found\\r\\n\"\n", "end_time": 1563354516.1571789}}
{"tag": "function", "depth": 1, "time": 1563354516.1601546, "data": {"name": "touch", "call_args": {"v": [192.0, 662.0], "kwargs": {"duration": 0.01}, "times": 1}, "start_time": 1563354515.792033, "traceback": "Traceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 40, in _install_and_setup\n    apk_path = self.adb.path_app(ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 1162, in path_app\n    raise AirtestError('package not found, output:[%s]' % output)\nairtest.core.error.AirtestError: 'package not found, output:[]'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\logwraper.py\", line 72, in wrapper\n    res = f(*args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\api.py\", line 259, in touch\n    try_log_screen()\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\logwraper.py\", line 72, in wrapper\n    res = f(*args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\cv.py\", line 97, in try_log_screen\n    screen = G.DEVICE.snapshot()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\android.py\", line 217, in snapshot\n    self.rotation_watcher.get_ready()\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\snippet.py\", line 104, in ready_func\n    method()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 61, in start\n    self._install_and_setup()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 42, in _install_and_setup\n    self.adb.install_app(ROTATIONWATCHER_APK, ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 568, in install_app\n    out = self.cmd(cmds)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 185, in cmd\n    raise DeviceConnectionError(stderr)\nairtest.core.error.DeviceConnectionError: \"error: device '76UBBL422HJF' not found\\r\\nerror: device '76UBBL422HJF' not found\\r\\nadb: connect error for write: device '76UBBL422HJF' not found\\r\\n\"\n", "end_time": 1563354516.1601546}}
{"tag": "info", "depth": 0, "time": 1563354516.1641226, "data": {"name": "Final Error", "traceback": "Traceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 40, in _install_and_setup\n    apk_path = self.adb.path_app(ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 1162, in path_app\n    raise AirtestError('package not found, output:[%s]' % output)\nairtest.core.error.AirtestError: 'package not found, output:[]'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"c:\\python\\lib\\site-packages\\airtest\\cli\\runner.py\", line 61, in runTest\n    exec(compile(code.encode(\"utf-8\"), pyfilepath, 'exec'), self.scope)\n  File \"C:\\Git\\gaodingAppTest\\pythonProject\\gaodingAppUIAutoTest\\air\\testSwitchRatioAndSavePhoto.air\\testSwitchRatioAndSavePhoto.py\", line 21, in <module>\n    poco( text=\"\u56fe\u7247\u6807\u8bb0\" ).click()\n  File \"c:\\python\\lib\\site-packages\\poco\\proxy.py\", line 23, in wrapped\n    return func(proxy, *args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\poco\\proxy.py\", line 334, in click\n    ret = self.poco.click(pos_in_percentage)\n  File \"c:\\python\\lib\\site-packages\\poco\\pocofw.py\", line 252, in click\n    ret = self.agent.input.click(pos[0], pos[1])\n  File \"c:\\python\\lib\\site-packages\\poco\\utils\\airtest\\input.py\", line 65, in click\n    touch(pos, duration=self.default_touch_down_duration)\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\logwraper.py\", line 72, in wrapper\n    res = f(*args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\api.py\", line 259, in touch\n    try_log_screen()\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\logwraper.py\", line 72, in wrapper\n    res = f(*args, **kwargs)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\cv.py\", line 97, in try_log_screen\n    screen = G.DEVICE.snapshot()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\android.py\", line 217, in snapshot\n    self.rotation_watcher.get_ready()\n  File \"c:\\python\\lib\\site-packages\\airtest\\utils\\snippet.py\", line 104, in ready_func\n    method()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 61, in start\n    self._install_and_setup()\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\rotation.py\", line 42, in _install_and_setup\n    self.adb.install_app(ROTATIONWATCHER_APK, ROTATIONWATCHER_PACKAGE)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 568, in install_app\n    out = self.cmd(cmds)\n  File \"c:\\python\\lib\\site-packages\\airtest\\core\\android\\adb.py\", line 185, in cmd\n    raise DeviceConnectionError(stderr)\nairtest.core.error.DeviceConnectionError: \"error: device '76UBBL422HJF' not found\\r\\nerror: device '76UBBL422HJF' not found\\r\\nadb: connect error for write: device '76UBBL422HJF' not found\\r\\n\"\n"}}
